src/braket/device_schema/standardized_gate_model_qpu_device_properties_v1.py

```diff
@@ -113,7 +113,7 @@ class OneQubitProperties(BaseModel):
 
     T1: CoherenceTime
     T2: CoherenceTime
-    oneQubitFidelity: List[Fidelity1Q]
+    oneQubitFidelity: list[Fidelity1Q]
```

Comment:
Thanks for adding the changes. 

Unrelated to the CR, it looks we have inconsistent doc-strings format all around, we have data types in some of them and not in others.

Reply:
Yeah, we should probably comb through the codebase and fix them all up at some point.